src/components/slider/slider.js

```diff
@@ -254,6 +353,29 @@ function SliderDirective($$rAF, $window, $mdAria, $mdUtil, $mdConstant, $mdThemi
       }
     }
 
+    function mouseDownListener() {
+      redrawTicks();
+
+      scope.mouseActive = true;
+      wrapper.removeClass('md-focused');
+
+      $timeout(function() {
+        scope.mouseActive = false;
+      }, 100);
```

Comment:
Why `100ms`? We generally try to avoid having a hard-coded timeout if possible. Maybe we could use `$mdUtil.nextTick()` instead?